specs/daito/Multi_cloud_yig_integration.md

```diff
@@ -0,0 +1,325 @@
+# Multi-Cloud feature enhancements
+
+**Author(s)**: [Shufang Zeng](https://github.com/sfzeng) [Neelam Gupta](https://github.com/neelamgupta1491) [Ashit Kumar](https://github.com/kumarashit) 
+
+## Summary
+
+OpenSDS provides multi-cloud management platform. The Goal of multi-cloud is to enable data autonomy for multi-cloud environments. Currently OpenSDS multi-cloud (Gelato) provides multiple features for Object Data Migration, Lifecycle and Management across multiple clouds like AWS, Azure, GCP, Huawei, IBM. Also, Gelato need to increase the multi-cloud supportability matrix. In the next release, OpenSDS will add some key features, add new on-prem Object storage i.e. YIG and integrate with YIG to enhance the multi-cloud features of OpenSDS.
+The purpose of this design is to add new features and analyse and facilitate the OpenSDS multi-cloud (Gelato) integration with YIG (Yet Another Index Gateway)
```

Comment:
nit: "add new features and analyse and facilitate..." -> "add new features, analyse and facilitate the...."
nit: The YIG acronym can be explained at first occurrence above.